src/plat/imx8m-evk/config.cmake

```diff
@@ -17,6 +18,10 @@ if(KernelPlatformImx8mq-evk OR KernelPlatformImx8mm-evk)
     else()
         fallback_declare_seL4_arch_default(aarch64)
     endif()
+    
+    if(KernelPlatformImx8mq-evk)
```

Comment:
Is this fixing a general problem with the i.MX8 board family setup that just popped up with you adding a new board now?

Reply:
The intention here is to introduce a configuration macro to capture that this is an i.MX8MQ based board. This has allowed us to simplify other configuration changes for settings that are shared across all boards that use this SoC.

This makes the config changes for this board simpler, and will greatly simplify the changes that would be needed for any future boards that use the same SoC.

Reply:
Ah ok, I get it. So this basically goes into the direction of what `src/plat/imx6/config.cmake` does. Just there all boards are in the same folder, while for i.MX8 we have separate folders now and `KernelARMPlatform` is used differently.

Reply:
That's right. I was looking to make the smallest change (without completely restructuring all of the existing iMX8 boards) that would allow for the greatest resume of setting across boards.